sensors/jira_sensor_for_apiv2.py

```diff
@@ -105,9 +105,9 @@ def _dispatch_issues_trigger(self, issue):
         payload = {}
         payload['project'] = self._project
         payload['id'] = issue.id
-        payload['expand'] = issue.raw['expand']
+        payload['expand'] = issue.raw.get('expand', None)
```

Comment:
In the payload_schema of this sensor, `expand` parameter requires string type. So I think `''` is better than `None` as a default value.